src/test/nodestore/DatabaseShard_test.cpp

```diff
@@ -978,6 +1090,93 @@ class DatabaseShard_test : public TestBase
         }
     }
 
+    std::string
+    ripemd160File(std::string filename)
```

Comment:
Same comment as earlier: please use the hasher.

Reply:
Done